### PR TITLE
Update MVG documentation

### PR DESCRIPTION
**Pull request by mountainsandcode**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15454

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Looks good, can be merged as soon as the parent PR gets merged.
Thank @mountainsandcode! 👍

**Review by frenck — DISMISSED** [review]

**Comment by DavidMStraub** [comment]
Perfect, thanks!

File: source/_components/sensor.mvglive.markdown [hunk 1]
```diff
@@ -55,8 +56,11 @@ sensor:
...
      - station: Sendlinger Tor
        lines: ['U2','U8']
+       number: 5
      - station: Scheidplatz
        products: ['U-Bahn']
        directions: '1'
 ```
-The first sensor will return S-Bahn departures to Munich Airport or Markt Schwaben that are at least 2 minutes away. The second sensor returns U2 and U8 departures from Sendlinger Tor while the third sensor returns all south-bound U-Bahn trains from Scheidplatz.
+The first sensor will return S-Bahn departures to Munich Airport or Markt Schwaben that are at least 2 minutes away. 
+The second sensor returns U2 and U8 departures from Sendlinger Tor and stores a total of 5 departures in attributes. To retrieve the time until the second departure, you would use states.sensor.entiy_name.attributes.departures[1].time.
```
Comment (MartinHjelmare) [inline source/_components/sensor.mvglive.markdown]: states.sensor.ENTITY_NAME.attributes.departures[1].time.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**